js/stringProcessing/getWords.js

```diff
@@ -2,6 +2,9 @@
 
 var stripTags = require( "../stringProcessing/stripHTMLTags.js" );
 var stripSpaces = require( "../stringProcessing/stripSpaces.js" );
+var removeSentenceTerminators = require( "../stringProcessing/removeSentenceTerminators.js" );
```

Comment:
you could shorten the requires by using `./` in stead of `../stringProcessing` since getWords is in the stringProcessing directory